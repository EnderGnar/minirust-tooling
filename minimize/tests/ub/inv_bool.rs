include!("../helper/transmute.rs");

fn main() { unsafe {
    let _b = transmute::<u8, bool>(2);
} }
