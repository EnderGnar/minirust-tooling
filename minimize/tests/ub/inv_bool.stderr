UB: load at type PlaceType { ty: Bool, align: Align { raw: Int(Small(8)) } } but the data in memory violates the validity invariant
